warehouse/api/http.go

```diff
@@ -127,6 +129,9 @@ func NewApi(
 
 func (a *Api) Start(ctx context.Context) error {
 	srvMux := chi.NewRouter()
+	srvMux.Use(
+		chiware.StatMiddleware(ctx, srvMux, a.statsFactory, "warehouse"),
```

Comment:
The 2nd argument doesn't seem to be used internall, @atzoum can we remove it from the kit?

Reply:
https://github.com/rudderlabs/rudder-go-kit/pull/173